src/content/toys.ts

```diff
@@ -32,17 +32,88 @@ function makeBlock() {
     overlaydiv.style.position = "fixed"
     overlaydiv.style.display = "block"
     overlaydiv.style.width = String(window.innerWidth)
-    overlaydiv.style.height = String(window.innerHeight)
-    overlaydiv.style.top = "0"
-    overlaydiv.style.left = "0"
-    overlaydiv.style.right = "0"
-    overlaydiv.style.bottom = "0"
-    overlaydiv.style.zIndex = "1000"
-    overlaydiv.style.opacity = "0.5"
+    overlaydiv.style.height = String(document.documentElement.scrollHeight)
+    overlaydiv.style.top = "0px"
+    overlaydiv.style.bottom = "0px"
+    overlaydiv.style.left = "0px"
+    overlaydiv.style.right = "0px"
+    overlaydiv.style.zIndex = "100"
+    overlaydiv.style.opacity = "0.8"
```

Comment:
I'd prefer that we made this configurable, e.g.

`function music(opts = {opacity: 0.5})`

as it makes `neo_mouse_mode` less usable. Which I realise is a niche concern...

Reply:
Configurable is not the right word - I don't mean `config.get`, I mean such that `drawable_mouse_mode` can set it to 0.8 when it is called

Reply:
If its only that, can we not just change the attribute in `make_drawable`? I've already done that for `style.position`.

Reply:
Sure, it doesn't make any difference to me : )